### ISSUE TITLE
I SCREWED UP...

### ISSUE THREAD

**Issue by MrDwarf-95** [darkautism/PS3TrophyIsGood#32]
Hacking the two Gran Turismo 5 online trophies I made a mistake and put the wrong month in one of them. Now that trophy is dated before I created my psn account and it was synchronized.

Opinions? Anyone in the same or similar situation?

Thanks.

**Comment by darkautism** [comment]
I think we can get throphy before you create PSN account.

**Comment by MrDwarf-95** [comment]
Yes, the problem is that these two trophies are online and require a connection to PSN.
